packages/fluentui/docs/src/examples/components/Loader/Variations/LoaderExampleSecondary.tsx

```diff
@@ -1,20 +1,26 @@
 import { Loader, Flex, Provider } from '@fluentui/react-northstar';
 import * as React from 'react';
 
-const LoaderExampleSecondary: React.FC = () => (
-  <Provider.Consumer
-    render={theme => (
-      <Flex
-        style={{
-          backgroundColor: theme.siteVariables.colorScheme.brand.background,
-          padding: 8,
-          width: 'fit-content',
-        }}
-      >
-        <Loader secondary />
-      </Flex>
-    )}
-  />
-);
+const LoaderExampleSecondary: React.FC = () => {
+  return (
+    <Provider.Consumer
+      render={theme => {
+        const themeBackgroundColor = theme.siteVariables.colorScheme.brand.background;
+        const siteColors = theme.siteVariables.colors;
+        return (
+          <Flex
+            style={{
+              backgroundColor: themeBackgroundColor !== siteColors.white ? themeBackgroundColor : siteColors.black,
```

Comment:
This does not look fancy, is there any other color what could be used to match all themes?

```suggestion
              backgroundColor: theme.siteVariables.colorScheme.brand.SOME_COLOR,
```

Reply:
Thank you Shift, I used theme.siteVariables.colorScheme.brand.backgroundActive